bin/picca_Pk1D.py

```diff
@@ -7,152 +7,367 @@
 from array import array
 import numpy as np
 import fitsio
+import os
 
 from picca import constants
 from picca.data import Delta
-from picca.pk1d import (compute_correction_reso, compute_pk_noise,
+from picca.pk1d import (compute_correction_reso,
+                        compute_correction_reso_matrix, compute_pk_noise,
                         compute_pk_raw, fill_masked_pixels, rebin_diff_noise,
                         split_forest)
 from picca.utils import userprint
+from multiprocessing import Pool
 
 
-def make_tree(tree, max_num_bins):
-    """Makes the ROOT tree to save the data
+def check_linear_binning(delta):
```

Comment:
Partly because I didn't ingest how to do this properly, yet. Partly because I forgot about the file that has all config options inside and not only the explicit ones. And partly because in principle this could be run on data created without delta_extraction as well (at least until all the pieces are actually fitting together and e.g. tests have been adapted)...

Reply:
I agree that it is better to keep it separate from delta_extraction. I do not know if the delta creation from others will necessarily be implemented in picca.

Reply:
makes sense